### PR TITLE
20 button

### PR DESCRIPTION
**Pull request by Amoodaa**
**Issue**: #20

## What did I do?
- Added storybooks/addon-knobs
- normalized more global styling [check this](https://github.com/yalla-coop/website-redesign/pull/34/commits/cd153f8e1dcc2071fceeb1bf3b7b768faac5d0b0) i staged the button file by mistake i think
- Styled the main button
- Added tons of props in the button

## How to test?
You can check the usage of the `primary` story by running `storybook`.
And you can test all the props with the custom button story.

### CODE REVIEW

File: src/components/elements/buttons/index.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  title,
+  hasArrow,
+  textColor,
+  backgroundColor,
+  onClick,
+  capitalize,
+  primary,
+}) => {
+  const bgColor = (primary && colors.primary) || backgroundColor
+  return (
```
Comment (fadeomar) [inline src/components/elements/buttons/index.js]: the onClick event is required you can add it like so : 
 ```suggestion
const Button = ({
  title,
  hasArrow,
  textColor,
  backgroundColor,
  onClick,
  capitalize,
  primary,
}) => {
  const bgColor = (primary && colors.primary) || backgroundColor
  const onClik = onClick || () => alert('you must add onClick event')
  return (
```

File: src/components/elements/buttons/index.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  capitalize: PropTypes.bool,
+  primary: PropTypes.bool,
+}
+
+Button.defaultProps = {
+  backgroundColor: colors.primary,
+  hasArrow: false,
+  capitalize: false,
+  primary: false,
+  onClick: () => {},
```
Comment (fadeomar) [inline src/components/elements/buttons/index.js]: I beleive the onClick must be requerd

File: src/components/elements/buttons/index.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  ${font};
+  font-size: ${size.xxs};
+  font-weight: 500;
+  letter-spacing: ${letterSpacing.medium};
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1rem 2rem;
+  border-radius: 5px;
+  border: none;
```
Comment (yosefanajjar) [inline src/components/elements/buttons/index.js]: I think you need to remove the outline and the cursor should be set to pointer here
```css
{
  outline: none;
  cursor: pointer;
}
```

Comment (Amoodaa) [reply src/components/elements/buttons/index.js]: done

**Comment by yosefanajjar** [comment]
@Amoodaa check out what happens when the build script runs there is an error that happens with the Netlify build 😃

**Pull request merged**